.github/workflows/scripts/install.sh

```diff
@@ -46,7 +46,7 @@ image:
   tag: "${TAG}"
 plugins:
   - name: pulpcore
-    source: pulpcore
+    source: pulpcore~3.11
```

Comment:
you probably need to run plugin_template again because this version didn't seem to be updated to the most recent change in the config.